lib/Dialect/TritonGPU/Transforms/DecomposeScaledBlocked.cpp

```diff
@@ -30,6 +30,9 @@ class DecomposeScaledBlocked : public OpRewritePattern<DotScaledOp> {
 
   LogicalResult matchAndRewrite(DotScaledOp scaledDotOp,
                                 PatternRewriter &rewriter) const override {
+    // TODO: add support for m/n packed formats.
+    if (!scaledDotOp.getLhsKPack() || !scaledDotOp.getRhsKPack())
+      return failure();
```

Comment:
Not for now, but we can support this simply by transposing the inputs upcasting and transposing the results before feeding into the dot.

Reply:
yes seems doable